CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ Added
 
 Changed
 
+- Change/deprecate `version` attribute to `audit_version` attribute
```

Comment:
It should be clear this is only for the audited resource revisions, not the audits themselves. Perhaps something like this?
```markdown
Rename audited resource revision `version` attribute to `audit_version` and deprecate `version` attribute

Reply:
Works for me. 👍